docs/install-docker-compose-quickstart.md

```diff
@@ -0,0 +1,105 @@
+# Docker Quickstart
+
+[Docker](https://www.docker.com/) is the quickest way to download and start developing with Nakama. By using Docker you are able to:
+
+* Install to a pristine environment
+* Easily install and run the [CockroachDB](https://www.cockroachlabs.com/) or [PostgreSQL](https://www.postgresql.org/) database for Nakama
+* Take snapshots, remove, and re-install Nakama without affecting your primary operating system
+* Enjoy a quick and simplified installation experience regardless of your OS
+
+Following this guide, you will use [Docker Compose](https://docs.docker.com/compose/) to quickly and easily define all the necessary services and run your local developement instance of Nakama.
+
+## Prerequisites
+
+Before proceeding ensure that you have [installed Docker Desktop](https://docs.docker.com/get-docker/).
+
+## Running Nakama
+
+1. Start by creating a directory where your Nakama server will sit, for example `Desktop/nakama`.
+2. In this folder create a `docker-compose.yml` file and open it using your preferred text editor.
+3. Heroic Labs provides two YML files for use: using either [CockroachDB](https://github.com/heroiclabs/nakama/blob/master/docker-compose.yml) or [PostgreSQL](https://github.com/heroiclabs/nakama/blob/master/docker-compose-postgres.yml) as the database.
```

Comment:
1. Can we use embed these resources on the page directly from GitHub, some type of embedding by any chance?
2. We should probably edit these to include a `volume` and load a custom config file (with a really simple config) which will significantly make following "## Configuration File" much easier to follow IMO.

Reply:
One option that showed up after some searching: https://www.npmjs.com/package/embedme
maybe @seanpackham knows some other options here?

Reply:
Shared a mkdocs extension with Gabe https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#snippets

Reply:
@mofirouz There is no immediately available solution for this right now (that wouldn't involve some level of duplication, defeating the purpose), but did discuss with Sean and it is something we will be able to address in v2 of the doc site.

I think let's merge this as-is now, and improve upon it in v2.